### ISSUE TITLE
Add action to encode component

### ISSUE THREAD

**Issue by GauthierPLM** [AntonShuvaev/jet-client-support#12]
### Is your feature request related to a problem?

Encoding URI components is very common when they contains special characters such as `/` to be used as variables or URI params, but JetClient currently does not have any to do so.

### Describe the solution you'd like

When right clicking on a string, add an option to encode the selected text, or the whole string if no text is selected.

**Comment by AntonShuvaev** [comment]
The URL is encoded automatically when you send a request. Did you have any issues with it?

**Comment by GauthierPLM** [comment]
Yes, one of my APIs take an S3 key as URI parameters, so `/{buckedId}/objects/{key}`. As this key contains `/`, the path is not properly processed because JetClient does not encode the / by default.

**Comment by AntonShuvaev** [comment]
Will add it to backlog. 

For now, you can use [String Manipulation plugin](https://plugins.jetbrains.com/plugin/2162-string-manipulation) with which you can select the string, open Actions, and type and select Encode URL or add a shortcut to it.

**Comment by Evenegas4414** [comment]
Hello, this pluggin is very helpful and good.
But I have a problem with the "auto-encode", when I try to use a endpoint and get information about a key, if the key contains a + sign, It is encode as blank space.
There can be an option to disable this feature?  "Auto-encoding" I mean.

**Comment by calteran** [comment]
+1 on disabling auto-encoding.  I have parameters that come pre-encoded from my API.  JetClient re-encodes the percent signs and breaks the downline request.

**Comment by AntonShuvaev** [comment]
I will add the option for disabling auto-encoding in the next version.
